src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -3539,13 +3539,13 @@ boolean group(Context context, DeclarationList declList) throws ParserException
         Context ctx = new Context(context);
         Token[] prefixes = {Token.CLASS, Token.INTERFACE, Token.__INTERFACE, Token.STRUCT, Token.UNION};
         for (Token token = tokens.get(); !token.match(Token.EOF); token = tokens.next()) {
-            if (token.match(prefixes)) {
+            if (token.match((Object[]) prefixes)) {
```

Comment:
This remove a compilation warning about ambiguous call between vararg and non-vararg.